frontend/Dockerfile.dev

```diff
@@ -0,0 +1,13 @@
+FROM node:18-alpine
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 5173
```

Comment:
We are running the React Frontend on `PORT 3000` so it would be better to change `5173` to `3000`